src/components/ChatList/ChannelItem.style.ts

```diff
@@ -8,7 +8,7 @@ export const channelItemStyles = StyleSheet.create({
     display: 'flex',
     flexDirection: 'row',
     paddingLeft: dimen.normalizeDimen(20),
-    backgroundColor: COLORS.white
+    backgroundColor: COLORS.almostBlack
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.white` to `COLORS.almostBlack` for the background color of the `ChannelItem` component might affect readability if the text color is not adjusted accordingly. Please ensure that the text color contrasts well with this new background color for better accessibility and readability.
```diff
-    backgroundColor: COLORS.white
+    backgroundColor: COLORS.almostBlack
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->